### ISSUE TITLE
number is somehow weird for CA Attorney General Xavier Becerra (800-952-5225)

### ISSUE THREAD

**Issue by piebob** [5calls/5calls#372]
For the Net Neutrality issue there is only one number listed for the CA Attorney General Xavier Becerra (800-952-5225). I'm calling from a cellphone in California, and I got the message "sorry that number is not available from your calling area". I have never gotten this message before and I can't find another number to reach the Attorney General. Can you list alternates on the website in the event that other people have this problem?

**Comment by nickoneill** [comment]
Yeah, we've seen this before. There are two numbers for Becerra, one works for some people and not others, the other works for some other set of people but not others. I can't figure out what the restriction is here in the first place for who needs which number.
